tests/test_pyclipper.py

```diff
@@ -246,8 +246,9 @@ def test_execute2(self):
 
     def test_clear(self):
         self.pc.Clear()
-        solution = self.pc.Execute(*self.default_args)
-        self.assertEqual(len(solution), 0)
+        with self.assertRaises(pyclipper.ClipperException):
+            solution = self.pc.Execute(*self.default_args)
+            self.assertEqual(len(solution), 0)
```

Comment:
If Execute raises an exception, it never reaches the following assertEqual, so that line should be redundant, no?

Reply:
You are right, I thought about keeping the assertEqual test with a try/except block, but that would be pointless since pyclipper would already have raised an exception. I just removed the redundant line. Any thoughts?